src/AppBundle/Committee/CommitteeCreatedEvent.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace AppBundle\Committee;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\Committee;
+
+class CommitteeCreatedEvent extends CommitteeEvent
```

Comment:
I thought about these names but I like the events to be named in past tense. Maybe the best name would be `CommitteeWasCreatedEvent` like in CQRS recommendations.